vendor/assets/javascripts/nested_resource.js

```diff
@@ -0,0 +1,113 @@
+var NestedResource = function(el, options) {
```

Comment:
Need to implement a namespace for all the JS classes. `Cmsify.NestedResource` is probably good.